bittide/src/Bittide/DoubleBufferedRam.hs

```diff
@@ -21,93 +22,85 @@ import Protocols.Wishbone
 import Bittide.SharedTypes hiding (delayControls)
 import Data.Typeable
 
-data InitialContent elements a where
-  NonReloadableV  :: Vec elements a -> InitialContent elements a
-  ReloadableV     :: Vec elements a -> InitialContent elements a
-  NonReloadableMb :: MemBlob elements (BitSize a) -> InitialContent elements a
-  ReloadableMb    :: MemBlob elements (BitSize a) -> InitialContent elements a
-  Undefined       :: (1 <= elements, KnownNat elements) => InitialContent elements a
+data ContentType n a
+  = Vec (Vec n a)
+  | Blob (MemBlob n (BitSize a))
+  | ByteVec (Vec (Regs a 8) (MemBlob n 8))
```

Comment:
Would `BlobVec` make more sense? When I read `ByteVec` further down below I first though it was a `Vec n (Bytes m)` and had to come back up to double check

Reply:
I agree, changed it